### DISCUSSION TITLE
REST Table sorting

### DISCUSSION THREAD

**Discussion by aloreaux** [Budibase/budibase#7551] [Help] [answered]
The default sort column works, but if I click on another column that is a number it sorts via alphabetical and not the number. If I choose the number as the default column it will sort by the number. But then trying to sort by the date or name of something still bases it on the number. Anyway to make it so all the columns can sort independently depending on their type?

I appear to be having a similar issue as this one:  https://github.com/Budibase/budibase/discussions/5757#discussion-4057406

Thanks!

**Comment by melohagan** [comment, marked as answer]
Hey @aloreaux 

Thanks for reporting. This is a known bug, but I did find a workaround: https://github.com/Budibase/budibase/issues/4804#issuecomment-1223818865
